vars/sectionDeployToAKS.groovy

```diff
@@ -86,18 +90,33 @@ def call(params) {
           }
 
           onFunctionalTestEnvironment(environment) {
-            stageWithAgent("Functional Test - AKS ${environment}", product) {
+            onPR {
+              if (!testLabels.contains('enable_full_functional_tests')) {
+                stageWithAgent("Functional Test - ${environment}", product) {
+                  testEnv(aksUrl) {
+                    pcr.callAround("functionalTest:${environment}") {
+                      timeoutWithMsg(time: 40, unit: 'MINUTES', action: 'Functional Test - AKS') {
+                        builder.functionalTest()
+                      }
+                    }
+                  }
+                }
+              }
+            }
+            stageWithAgent('Functional test (Full)', product) {
```

Comment:
this needs to be gated around an if statement

Reply:
Updated.